TempleEight/blog/2019-05-30-welcome.md

```diff
@@ -1,15 +0,0 @@
----
-id: welcome
-title: Welcome
-author: Lewis Bell
-author_title: Full Stack Ninja @ TempleEight
```

Comment:
@Lewiky is going to cry

Reply:
I know, it listed me as something more than just the intern/qa lead :(

Reply:
Cause I've been promoted now to DOCS LEAD